pre-commit.sh

```diff
@@ -1,5 +1,39 @@
-#!/bin/bash -eux
-yapf --in-place --recursive .
+#!/bin/bash -ux
+
+base_remote="$(1:-origin)"
```

Comment:
Fragment of `pre-commit.sh` output:
```
++ 1:-origin
./pre-commit.sh: line 3: 1:-origin: command not found
+ base_remote=
```
Is it ok? The same problem occured for next two lines in the script.